Library/ViewModels/CommentsViewModel.swift

```diff
@@ -221,9 +242,21 @@ public final class CommentsViewModel: CommentsViewModelType,
       .map { ($1, $0) }
       .flatMap(retryCommentProducer)
 
-    self.showLoadingIndicatorInFooterView = Signal
-      .combineLatest(isCloseToBottom, self.beginOrEndRefreshing)
-      .map(second >>> isTrue)
+    let footerViewActivityState = Signal
+      .combineLatest(isCloseToBottom, isLoading)
+      .filter(second >>> isTrue)
```

Comment:
Can you remind me what the >>> operator does again?

Reply:
Evaluates the composition of these two functions, same as saying:

```swift
.filter { isTrue($0.1) }
```

`second` is a function to pick out the second part of a tuple.

Reply:
in my understanding I think it says, if a function take input type A and return type B, and another function takes input of type B and returns different type C, you can compose them using the `>>>` operator.

```
func toString(_  value: Int) -> String { return String(value) } 
func split(_  value: String) -> [String] { return value.split } 
```

You can compose this by saying 

`toString >>> split`